frontend/src/tests/pages/Rider/RiderCreatePage.test.js

```diff
@@ -0,0 +1,127 @@
+import { render, waitFor, fireEvent } from "@testing-library/react";
```

Comment:
Please do an incremental global search / replace one at a time on `Rider` and see for each one, whether it should be `RideRequest`.

Also: what's in the backend?  Is it Rider?  or RideRequest?  Do we need to fix it there also?